packages/editor/src/components/post-card-panel/index.js

```diff
@@ -83,8 +93,19 @@ export default function PostCardPanel( { actions } ) {
 					className="editor-post-card-panel__title"
 					weight={ 500 }
 					as="h2"
+					lineHeight={ '20px' }
 				>
 					{ title ? decodeEntities( title ) : __( 'No Title' ) }
+					{ isFrontPage && (
+						<span className="editor-post-card-panel__title-badge">
```

Comment:
Is this badge something that can become a generic UI components to move to the components package? We're using it in multiple places now.

Reply:
Yes, probably so. Also in use for anchor tags in list view and within data views. 
 
![CleanShot 2024-05-29 at 09 46 40](https://github.com/WordPress/gutenberg/assets/1813435/3c28cad0-7024-415f-a344-f3257ff23c92)
![CleanShot 2024-05-29 at 09 47 05](https://github.com/WordPress/gutenberg/assets/1813435/8a612df7-7860-444b-a7b0-5a4889aae12e)

Reply:
Yes please https://github.com/WordPress/gutenberg/issues/61106